### PR TITLE
Update 2023-12-06-tsc.md

### PR DESCRIPTION
**Pull request by asamborski**
Minutes from December 6, 2023 TSC meeting

### CODE REVIEW

File: meeting-tsc/2023-12-06-tsc.md [hunk 1]
```diff
@@ -3,9 +3,51 @@
...
+
+Ethan is chair by unanimous agreement by TSC members. 
+
+## Discuss How We Document and Standardize OpenPubkey
+
+RFC is a nice format, but there are concerns it is too formal to serve the community as a whole. 
+
+The goal is to give the community a structured way to interact with the design of OPK rather than just the code. 
+
+For now, we think a Markdown file in GitHub makes sense (within the `openpubkey/openpubkey` repo) so folks can make PRs against it. We would like to adopt the RFC that defines usage of "must," "should," and "may." Diagrams in Markdown can use Mermaid or if a more complex diagram is necessary, can always embed an image.
```
Comment (jonnystoten) [inline meeting-tsc/2023-12-06-tsc.md]: [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt) is what I was thinking of!

Comment (jonnystoten) [inline meeting-tsc/2023-12-06-tsc.md]: I mentioned https://github.com/in-toto/attestation/tree/main/spec/v1 and https://github.com/secure-systems-lab/dsse/blob/master/protocol.md as nice examples of specs in markdown on Github.

**Review by jonnystoten — APPROVED** [review]

File: meeting-tsc/2023-12-06-tsc.md (hunk 1 above)
Comment (EthanHeilman) [reply meeting-tsc/2023-12-06-tsc.md]: DSSE Protocol looks really nice. It would be a nice example to follow

**Review by EthanHeilman — APPROVED** [review]

File: meeting-tsc/2023-12-06-tsc.md (hunk 1 above)
Comment (asamborski) [reply meeting-tsc/2023-12-06-tsc.md]: Added this in! [debb349](https://github.com/openpubkey/community/pull/7/commits/debb34957cc01b09f022a0671100d1705480c41d)

Comment (asamborski) [reply meeting-tsc/2023-12-06-tsc.md]: Added these in. [debb349](https://github.com/openpubkey/community/pull/7/commits/debb34957cc01b09f022a0671100d1705480c41d)

**Pull request merged**